File: benchmark/PHP 7.0/冒泡排序.php
<?php

function bubble_sort(&$array) {
	$size = count($array);
	for ($i=0;$i<$size;$i++) {
		$k = $size-1-$i;
		for ($j=0;$j<$k;$j++) {
			if ($array[$j] < $array[$j+1]) {
				$temp = $array[$j];
				$array[$j] = $array[$j+1];
				$array[$j+1] = $temp;
			}
		}
	}
}

$starttime = microtime(true);

for ($c=0;$c<100000;$c++)
{
	$array = array(0,1,2,3,4,5,6,7,8,9);
	bubble_sort($array);
}
print_r($array);

echo microtime(true)-$starttime."\n";

File: benchmark/CZL 1.0/冒泡排序.c
bubble_sort(&a) 
{
	size = #a
	for i in (0, size-1)
	{
		k = size-2-i
		for j in (0, k)
			if (a[j] < a[j+1])
				a[j] >< a[j+1]
	}
}

main
{
	var a
	for i in (0, 100000)
	{
		a = [0,1,2,3,4,5,6,7,8,9]
		bubble_sort(a)
	}
	echo(a)
	echo(clock())
}